### DISCUSSION TITLE
sql_default with nullable false

### DISCUSSION THREAD

**Discussion by tobymao** [collerek/ormar#306] [Q&A]
I tried doing sql_default: 'now()' and nullable false, but pydantic complains that i'm missing a required field. i would have expected that with sql_default it would be optional to pass in that column into the model. am i missing something?

**Comment by collerek** [comment]
There is no `sql_default` parameter :)
You need to use either `default` with datetime or server_default with sqlalchemy func.now - both samples are in the docs.

**Reply by tobymao** [reply]
that's what i meant, it's server_default, still the same problem, i'll file an issue
